config/optimize

```diff
@@ -49,7 +51,7 @@ HOST_CXXFLAGS="$HOST_CXXFLAGS -Wno-format-security"
     fi
 
   # default multiarch support
-    case "`uname -m`" in
+    case "${MACHINE_HARDWARE_NAME}" in
```

Comment:
In fact, once we have determined `HOST_LIBDIR` it should be possible to export it avoid all of this processing in subsequent passes. Will update.